src/styles/container.scss

```diff
@@ -0,0 +1,20 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, auto));
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  &__block {
+    display: flex;
+    height: 300px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  @for $i from 5 through 0 {
```

Comment:
One little moment🙂. If you already use function, maybe the better practice in common case is to add number for each inner element of container not at index.html but with "::after" pseudo element setting it "content" value to "#{6 - $I}" (for this case). It can be useful if number of elements will increase.